add a new TWR doc (#43)
ROvACC/docs: training/TOWER.md

# TOWER CONTROLLER

**  **

Approved by George Enciu, vACC Director

**  **

Created by Teodor G Petrică, OPS

**  **

## 1. Position overview

The tower is generally responsible for all movements on the runways and within the control zone (CTR). It also determines the direction of operation at the airport. Different or additional tasks are described in the respective airport SOP.
 

> In addition to the safe handling of air traffic, it is particularly important to work efficiently when there is a high volume of traffic, as is often the case on the network at the moment! The pilot principle also applies here: "Safe, orderly and expeditious", i.e. first and foremost safe, but immediately afterwards orderly and smoothly (efficient).

Reviewer: ```suggestion
In addition to safely handling air traffic, it is essential to work efficiently when there is a high volume of traffic. The pilot principle also applies here: "Safe, orderly and expeditious", i.e. first and foremost safe, but immediately afterward orderly and smoothly (efficient).
```

Aircraft prefer to take off and land against the wind. The name of the runway indicates the direction in which it is pointing according to the heading. Runway 08L in Bucharest Otopeni therefore points 079° towards the east. The opposite direction, runway 26R, is rotated by 180° and points 259°, which is approximately towards the west. The wind direction in the METAR indicates the direction from which the wind is coming. If the wind is coming from 260° (west), for example, runway 26R is used to be able to take off and land with a headwind. If the wind is not coming directly from the direction in which a runway is pointing, the one where the headwind component is greatest is used.

The active runway(s) are published via the ATIS, whenever this station is available.

_Remarks_

The **operating direction** should not be confused with the **operating mode**. 

The operating mode is a term that describes how the selected runways are controlled. Each airfield has a variety of options. In addition to standard operation, an airfield can be operated for Low Visibility Operations (LVO), for example, in order to maintain flight operations even in bad weather. 

> At larger airfields with parallel runways, there are several possible ways of using them. Under certain conditions, the runways can be used in "Parallel Dependent", in which, for example, radar separation from the parallel runway must be maintained on the approach.

Reviewer: Better refer to the only parallel runway airfield in RO and define it as depended than generalise it
